### PR TITLE
Complete basic results and card component

### PR DESCRIPTION
**Pull request by priscillaapaxton**
# PR Template

### Types of changes made?

- [ ]  testing
- [ ]  documentation
- [x]  javascript
- [x]  functionality

### What does this PR do?

- Adds functionality to rendering search results
- Updates JS for card and class components

### Relevant Tickets?

- Finishes tickets for Card and Results components

### Questions/Comments/Relevant Screenshots?
both components still need styling

### CODE REVIEW

**Comment by hjawad22** [comment]
Everything looks good, thanks, Priscilla!

File: src/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './Card.css'
+
+const Card = ({ src, alt, name, location, category }) => {
```
Comment (tialaaa) [inline src/Card/Card.js:4]: Nice destructuring in this component!

File: src/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <p className='card-name'>{name}</p>
+        <p className='card-location'>{location}</p>
+        <p className='card-category'>{category}</p>
+      </div>
+    </div>
+  )
+}
+
+export default Card
+
```
Comment (tialaaa) [inline src/Card/Card.js:20]: Need to add propTypes for this component.

File: src/Results/Results.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+      <div className='results-container'>
+        <Card {...place} />
+      </div>
+    )
+  }))
+
+}
+
+export default Results
```
Comment (tialaaa) [inline src/Results/Results.js:16]: Missing propTypes here too.

**Pull request merged**